### PR TITLE
Notifications: Permanent option and close button

### PR DESCRIPTION
**Pull request by Nuckyz**
Also doesn't depend on FormTitles anymore

### CODE REVIEW

**Comment by Vendicated** [comment]
screenshot please

**Comment by Nuckyz** [comment]
> screenshot please

image

File: src/api/Notifications/styles.css [hunk 1]
```diff
@@ -22,17 +22,40 @@
     gap: 1.25rem;
 }
 
+.vc-notification-content {
+    width: 100%;
```
Comment (Vendicated) [inline src/api/Notifications/styles.css:26]: isn't this obsolete?

Comment (Nuckyz) [reply src/api/Notifications/styles.css:26]: nope

File: src/api/Notifications/styles.css [hunk 2]
```diff
@@ -22,17 +22,40 @@
...
+}
+
+.vc-notification-header {
+    display: flex;
+    justify-content: space-between;
+}
+
+.vc-notification-title {
+    color: var(--header-primary);
+    font-size: 16px;
```
Comment (Vendicated) [inline src/api/Notifications/styles.css:36]: using px as font size is bad, either use `large(r)` or something like % or em/rem

File: src/api/Notifications/styles.css [hunk 3]
```diff
@@ -22,17 +22,40 @@
...
+    font-weight: 600;
+    line-height: 20px;
+    text-transform: uppercase;
+}
+
+.vc-notification-close-btn {
+    color: var(--interactive-normal);
+    opacity: 0.5;
```
Comment (Vendicated) [inline src/api/Notifications/styles.css:44]: you should probably transition the colour too

File: src/api/Notifications/NotificationComponent.tsx [hunk 4]
```diff
@@ -74,14 +75,33 @@ export default ErrorBoundary.wrap(function NotificationComponent({
...
+                                onClose!();
+                            }}
+                        >
+                            <svg
+                                className="vc-notification-close-btn"
+                                width="24"
+                                height="24"
+                                viewBox="0 0 24 24"
+                            >
+                                <path fill="currentColor" d="M18.4 4L12 10.4L5.6 4L4 5.6L10.4 12L4 18.4L5.6 20L12 13.6L18.4 20L20 18.4L13.6 12L20 5.6L18.4 4Z" />
```
Comment (Vendicated) [inline src/api/Notifications/NotificationComponent.tsx:94]:
```suggestion
								<title id="vc-notification-dismiss-title">Dismiss Notification</title>
                                <path fill="currentColor" d="M18.4 4L12 10.4L5.6 4L4 5.6L10.4 12L4 18.4L5.6 20L12 13.6L18.4 20L20 18.4L13.6 12L20 5.6L18.4 4Z" />
```

File: src/api/Notifications/NotificationComponent.tsx [hunk 5]
```diff
@@ -74,14 +75,33 @@ export default ErrorBoundary.wrap(function NotificationComponent({
...
+                                e.preventDefault();
+                                e.stopPropagation();
+                                onClose!();
+                            }}
+                        >
+                            <svg
+                                className="vc-notification-close-btn"
+                                width="24"
+                                height="24"
+                                viewBox="0 0 24 24"
```
Comment (Vendicated) [inline src/api/Notifications/NotificationComponent.tsx:92]:
```suggestion
                                viewBox="0 0 24 24"
                                role="img"
                                aria-labelledby="vc-notification-dismiss-title"
```

**Review by Vendicated — CHANGES_REQUESTED** [review]

**Comment by Nuckyz** [comment]
done

**Review by Vendicated — APPROVED** [review]

**Pull request merged**